components/src/dynamo/vllm/handlers.py

```
@@ -72,6 +72,66 @@
+        if self._first_token_received:
+            await self._engine_client.abort(self._request_id)
+            logger.debug(
+                f"Deferred abort: first token already received, "
+                f"aborting request {self._request_id} now"
+            )
+        else:
+            logger.debug(
+                f"Deferred abort: first token not received for request "
+                f"{self._request_id}, spawning background task"
+            )
+            asyncio.create_task(self._wait_and_abort())
```

Ruff (0.15.11)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)